Automattic/sensei
# Add Student completes lesson email

Resolves #6473 

## Proposed Changes
* Add Student completes Lesson email for teacher.
* Fix: the namespace for email generators.
* Fix: pass patterns directly into sender and call them without a call to email customization.

## Testing Instructions


1. Enable `email_customization` feature flag.
2. Re-create emails (Sensei LMS -> Tools -> Re-create emails).
3. Check the `[student:displayname] completed [lesson:name]` email exists and has content.
4. Create a student and a course with at least two lessons.
5. As a student, go to the course and complete lessons. 
6. In case of only two lessons, it is expected to get one `student-completes-lesson` email. When the second lesson is completed the teacher gets the email that the course is completed.

## Screenshots
<img width="1680" alt="CleanShot 2023-03-04 at 21 28 39@2x" src="https://user-images.githubusercontent.com/329356/222908080-26221210-3d20-46bb-89ce-25a2ff62d41d.png">
<img width="1388" alt="CleanShot 2023-03-04 at 21 29 02@2x" src="https://user-images.githubusercontent.com/329356/222908087-162cc0fd-a881-44e3-b35e-0dfbbfb6c298.png">


## Pre-Merge Checklist
